tests/board-detection.spec.js

```diff
@@ -0,0 +1,55 @@
+var test = require('tape');
+var proxyquire = require('proxyquire');
+
+test('pid board detection', function(t) {
+  t.plan(2);
+  var Avrgirl = proxyquire('../avrgirl-arduino', {
+    'serialport': {
+      list: function(callback) {
+        callback( null, [
+          { comName: '/dev/cu.sierravsp', manufacturer: '', serialNumber: '',
+            pnpId: '', locationId: '', vendorId: '', productId: '' },
+          { comName: '/dev/cu.Bluetooth-Incoming-Port', manufacturer: '',
+            serialNumber: '', pnpId: '', locationId: '', vendorId: '',
+            productId: '' },
+          { comName: '/dev/cu.usbmodem1421', manufacturer: 'Arduino (www.arduino.cc)',
+            serialNumber: '55432333038351F03170', pnpId: '', locationId: '0x14200000',
+            vendorId: '0x2341', productId: '0x0043' }
+        ]);
+      },
+      SerialPort: require('./helpers/mockSerial').SerialPort
+    }
+  });
+  var avrgirl = new Avrgirl({
+    board: 'uno',
+  });
+  t.notOk(avrgirl.options.port, 'no port was passed in');
```

Comment:
does this particular assert look confusing when you run the tests?
Ie. Does it show up as:

:white_check_mark: no port was passed in

and is it a check to make sure that the test setup was created properly, or is it asserting something about avrgirl-arduino itself when it instantiates?